### DISCUSSION TITLE
Lost Domains for Deleting a Team

### DISCUSSION THREAD

**Discussion by MicaelPiresCom** [vercel/community#6593] [Help] [answered]
### Summary

Hello.

I had a PRO team which I have just deleted to stop the monthly subscription, thinking it would automatically send the associated domains to my HOBBY account.
I lost access to my two domains, sharadox.com and cimenu.com.
I also can't email the support, as the DNS configuration pointing to my Google Workspace service don't work.
If there was a warning, I may have missed it.
Is there any way of recovering my domains on my personal account please?
Feel free to ask for any additional information needed.

### Example

Very similar situation: https://github.com/orgs/vercel/discussions/3389

### Steps to Reproduce

1. Create a PRO team.
2. Transfer in the domains.
3. Delete team.
4. Lost domains.

**Comment by amyegan** [comment]
Hey @MicaelPiresX. I need to find your personal account so I can check what steps are needed to recover your domains. If you can share your vercel username or a deployment ID, I can use one of those to locate your Vercel account.

**Comment by MicaelPiresCom** [comment]
Hi, @amyegan, thanks for the quick response. micaelpires is my username. vercel.com/micaelpires

**Reply by amyegan** [reply, marked as answer]
Thanks, Miceal. You should be able to re-add this to your Hobby account now via the "Add" button from the Domains page: https://vercel.com/dashboard/domains

Please give that a try and let me know whether or not it works for you

**Reply by amyegan** [reply]
If you run into trouble adding it to your Hobby team, you can also try adding directly to your personal account at https://vercel.com/account/domains

**Reply by MicaelPiresCom** [reply]
Thank you VERY much for your support, domains recovered.
